src/system_monitor_top.erl

```diff
@@ -14,7 +14,7 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------------------
 %%% @doc
-%%% Collect Erlang process statitics and push it to Kafka
+%%% Collect Erlang process statitics and push it to Postgres
```

Comment:
nit1: s/statitics/statistics/g
nir2: here and elsewhere the comments have s/Kafka/Postgres/, but the destination is configurable with PG only being the built-in default; should it say "to the configured destination" or something like that?